### PR TITLE
Plugins cleanup

### PR DESCRIPTION
**Pull request by vxsx**
there could also be some savings about traversing the dropdown but this will be addressed later
there also doesn't seem to be much need of passing subnav around however there are no significant wins eliminating that

### CODE REVIEW

File: cms/static/cms/js/modules/cms.plugins.js [hunk 1]
```diff
@@ -7,21 +7,29 @@
 
     // TODO move out to separate module CMS-276
     var KEYS = {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: didn't we want to do that somewhere globally?

File: cms/static/cms/js/modules/cms.plugins.js [hunk 2]
```diff
@@ -627,53 +649,27 @@
...
-                if ($(window).height() + $(window).scrollTop() - nav.offset().top - dropdown.height() <= 10 &&
-                    nav.offset().top - dropdown.height() >= 0) {
-                    dropdown.css('top', 'auto');
-                    dropdown.css('bottom', offset);
-                    // if parent is within a plugin, add additional offset
-                    if (dropdown.closest('.cms-draggable').length) {
-                        dropdown.css('bottom', offset - 1);
-                    }
+                if (this.ui.window.height() + this.ui.window.scrollTop() -
+                    dropdown.offset().top - dropdown.height() <= 10 && nav.offset().top - dropdown.height() >= 0) {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: too long?

File: cms/templates/cms/toolbar/placeholder.html [hunk 3]
```diff
@@ -6,21 +6,16 @@
...
-        'plugin_language': '{{ language }}',
-        'plugin_restriction': [{% for module in allowed_plugins %}"{{ module}}"{% if not forloop.last %}, {% endif %}{% endfor %}],
-        'urls': { {% language request.toolbar.toolbar_language %}
-            'add_plugin': '{{ placeholder.get_add_url }}',
-            'copy_plugin': '{{ placeholder.get_copy_url }}'
-        } {% endlanguage %}
-    });
-});
-})(CMS.$);
+CMS._plugins.push(['cms-placeholder-{{ placeholder.pk|unlocalize }}', {
```
Comment (FinalAngel) [inline cms/templates/cms/toolbar/placeholder.html]: nice work

File: cms/static/cms/js/modules/cms.plugins.js (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/js/modules/cms.plugins.js]: yep, going to move them all away together in a separate PR

File: cms/static/cms/js/modules/cms.plugins.js (hunk 2 above)
Comment (vxsx) [inline cms/static/cms/js/modules/cms.plugins.js]: not really

File: cms/templates/cms/toolbar/placeholder.html (hunk 3 above)
Comment (vxsx) [inline cms/templates/cms/toolbar/placeholder.html]: actually let's get rid of unnecessary quotes as well

**Comment by FinalAngel** [comment]
LGTM
